src/mlpack/methods/kmeans/kill_empty_clusters.hpp

```diff
@@ -51,13 +51,17 @@ class KillEmptyClusters
       const size_t emptyCluster,
       const arma::mat& /* oldCentroids */,
       arma::mat& newCentroids,
-      arma::Col<size_t>& /* clusterCounts */,
+      arma::Col<size_t>& clusterCounts,
       MetricType& /* metric */,
       const size_t /* iteration */)
   {
-    // Kill the empty cluster.
-    newCentroids.col(emptyCluster).fill(DBL_MAX);
-    return 0; // No points were changed.
+    // Remove the empty cluster.
+    if (emptyCluster < newCentroids.n_cols)
```

Comment:
Looks like the brace placement and indentation is off, can you take a look at: https://github.com/mlpack/mlpack/wiki/DesignGuidelines#brace-placement.